README.md

```diff
@@ -1 +1,72 @@
-# HelloMicroverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse 
+
+> This is my first project. Setting up a "Hello world" repository to practice essential git skills
+
+![screenshot](screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With 
+
+- html
+- css 
+
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] [OPTIONAL] Please do not leave empty these parts. If you feel they are not essential in this project you are free to delete them.